Makefile

```
@@ -33,6 +34,16 @@ distclean: clean
 	-rm -f lib/$(KOS_ARCH)/*
 	-rm -f addons/lib/$(KOS_ARCH)/*
 
+docs:
+	cd $(KOS_BASE)/doc; \
```

```suggestion
	doxygen $(KOS_BASE)/doc/Doxyfile
```

Haven't test this but can we do this way instead of changing directory?

---

Yes, this way works and is better.